models/intel/person-detection-0200/description/person-detection-0200.md

```diff
@@ -8,24 +8,20 @@ prior boxes for 256x256 resolution.
 
 ## Example
 
-![](./person-detection-0100.png)
+![](./person-detection-0200.png)
 
 ## Specification
 
 | Metric                          | Value                                     |
 |---------------------------------|-------------------------------------------|
-| AP                              | 82.52% (internal test set)                |
-| Pose coverage                   | Standing upright, parallel to image plane |
-| Support of occluded persons     | YES                                       |
-| Occlusion coverage              | <50%                                      |
-| Min person height               | 100 pixels (on 1080p)                     |
+| AP @ [ IoU=0.50:0.95 ]          | 0.244 (internal test set)                 |
```

Comment:
@Ilya-Krylov please explain changes in metric. Now it looks like accuracy drop.

Reply:
@eizamaliev metric for accuracy measurement changed (was mAP with single large threshold 0.5, now coco API for accuracy measurement used. I guess it is because in OTE for model training pipeline based mmdetection, where coco embedded in validation. In future it will be easy to maintain models and accuracy if we migrate on the same metrics.